packages/flutter_driver/test/src/real_tests/extension_test.dart

```diff
@@ -543,6 +543,72 @@ void main() {
     expect(await getOffset(OffsetType.center), const Offset(40 + (100 / 2), 30 + (120 / 2)));
   });
 
+  testWidgets('getText', (WidgetTester tester) async {
+    await silenceDriverLogger(() async {
+      final FlutterDriverExtension extension = FlutterDriverExtension((String arg) async => '', true);
+
+      Future<String> getTextInternal(SerializableFinder search) async {
+        final Map<String, String> arguments = GetText(search, timeout: const Duration(seconds: 1)).serialize();
+        final Map<String, dynamic> result = await extension.call(arguments);
+        if (result['isError'] as bool) {
+          return null;
+        }
+        return GetTextResult.fromJson(result['response'] as Map<String, dynamic>).text;
+      }
+
+      await tester.pumpWidget(
+          MaterialApp(
+              home: Scaffold(body:Column(
+                key: const ValueKey<String>('column'),
+                children: <Widget>[
+                  Text('Hello1', key: ValueKey<String>('text1')),
```

Comment:
There are analyzer warnings in this file:
https://cirrus-ci.com/task/5248086624698368?command=main#L29